src/lapack/svddc.rs

```diff
@@ -0,0 +1,69 @@
+use lapacke;
+use num_traits::Zero;
+
+use crate::error::*;
+use crate::layout::MatrixLayout;
+use crate::types::*;
+use crate::svddc::UVTFlag;
+
+use super::{SVDOutput, into_result};
+
+pub trait SVDDC_: Scalar {
```

Comment:
I could for instance merge this with the existing SVD_ trait, and change the trait to provide another function (svddc, or sdd). This may be preferred.

Or alternatively a more complicated way to invoke the existing svd that switches things around depending.

Reply:
I'm planning to refactoring these basis of `Lapack` traits (`SVD_` and others) into single trait, but it should be another PR. Your implementation is enough match to current impls, and I will accept it.